ci.sh

```diff
@@ -7,5 +7,12 @@ if [[ $result ]] ; then
   exit 1
 fi
 
+result=$(python -m isort -rc . | grep -v "Skipped")
+if [[ $result ]] ; then
```

Comment:
what do we check here?

Reply:
Checking if unordered imports aren't remained

Reply:
shouldn't we check the exit code instead (`$?`) ?

Reply:
I've checked its behavior at first, but unfortunately both cases(ordered, unordered) retrun `0`.

Reply:
how about https://stackoverflow.com/questions/2480584/how-do-i-use-a-file-grep-comparison-inside-a-bash-if-else-statement ?

Reply:
Will look into again.
At least I found  below issue and codes.

https://github.com/timothycrosley/isort/issues/423
https://github.com/timothycrosley/isort/blob/d093f97c1d26854fbb082e43cfae8a062dacda56/isort/main.py#L382

Reply:
I could do as expected by using ` --check-only` opt for `isort` :)